Account/FxAClient10.swift

```diff
@@ -341,6 +378,63 @@ open class FxAClient10 {
 
         return makeRequest(mutableURLRequest, responseHandler: FxADevice.fromJSON)
     }
+    
+    open func oauthAuthorize(withSessionToken sessionToken: NSData, keyPair: RSAKeyPair, certificate: String) -> Deferred<Maybe<FxAOauthResponse>> {
+        let audience = self.getAudience(forURL: self.oauthURL)
+        let assertion = JSONWebTokenUtils.createAssertionWithPrivateKeyToSign(with: keyPair.privateKey,
+                                                                              certificate: certificate,
+                                                                              audience: audience,
+                                                                              issuer: "127.0.0.1",
```

Comment:
Can you point to the document that describes the issuer to use?

Reply:
The issuer value you're specifying here, ultimately winds up in the "iss" field of the assertion payload here:

  https://github.com/mozilla-mobile/firefox-ios/blob/master/FxA/FxA/JSONWebTokenUtils.m#L69

I don't think there's any component of the system that looks for such "iss" field in the assertion.  Whatever value you specify here is going to be completely ignored, and I'd recommend removing it entirely to reduce potential future confusion.

(We do have a lot of things that check the "iss" field on the underlying certificate, but that's set by the server when it signs your public key and the client has no control over it.)

Reply:
That sounds good @rfk. I'll add some documentation stating that.